go-fuzz-build/main.go

```diff
@@ -58,10 +58,34 @@ func makeTags() string {
 // that clients can then modify and use for calls to go/packages.
 func basePackagesConfig() *packages.Config {
 	cfg := new(packages.Config)
-	cfg.Env = append(os.Environ(), "GO111MODULE=off")
+
+	// Preliminary modules support: note that we do not set 
+	// GO111MODULE here in order to respect any GO111MODULE setting by the user
+	// as we are finding dependencies. Note, however, that we are
+	// still setting up a GOPATH to build, so we later will force 
+	// GO111MODULE to be off when building so that we are in GOPATH mode.
+	// If the user has not set GO111MODULE, the meaning here is
+	// left up to cmd/go (defaulting to 'auto' in Go 1.11-1.13,
+	// but likely defaulting to 'on' at some point during Go 1.14
+	// development cycle).
+	// Also note that we are leaving the overall cfg structure
+	// in place to support future experimentation, etc.
+	cfg.Env = os.Environ()
```

Comment:
I'd assume this is the default

Reply:
See other `cfg.Env = os.Environ()` comment below.